### ISSUE TITLE
[3.0.3] Hard start time events stop current event

### ISSUE THREAD

**Issue by deltecent** [ElvishArtisan/rivendell#476]
When a hard start time is reached in the log, and the cart does not have any playable cuts, the current playing event is stopped abruptly and the event after the hard start time event is started.
```
Aug 15 12:57:09 maxsrv caed: LoadPlayback  Card: 0  Stream: 0  Name: /var/snd/011707_001.wav  Handle: 173
Aug 15 12:57:09 maxsrv rdairplay: started audio cart: Line: 410  Cart: 11707  Cut: 1 Pos: 0  Card: 0  Stream: 0  Port: 1
Aug 15 12:57:09 maxsrv caed: PlaybackPosition - Card: 0  Stream: 0  Pos: 624  Handle: 173
Aug 15 12:57:09 maxsrv caed: Play - Card: 0  Stream: 0  Handle: 173  Length: 267096  Speed: 100000  Pitch: 0
Aug 15 12:57:09 maxsrv rdairplay: finished event: Line: 409  Cart: 30541  Cut: 1 Card: 0  Stream: 1  Port: 0
Aug 15 12:57:09 maxsrv caed: UnloadPlayback - Card: 0  Stream: 1  Handle: 172
Aug 15 13:00:00 maxsrv rdairplay: event failed: Line: 418  Cart: 60003
Aug 15 13:00:00 maxsrv caed: LoadPlayback  Card: 0  Stream: 1  Name: /var/snd/010329_001.wav  Handle: 174
Aug 15 13:00:00 maxsrv rdairplay: started audio cart: Line: 419  Cart: 10329  Cut: 1 Pos: 0  Card: 0  Stream: 1  Port: 1
Aug 15 13:00:00 maxsrv caed: PlaybackPosition - Card: 0  Stream: 1  Pos: 96  Handle: 174
Aug 15 13:00:00 maxsrv caed: Play - Card: 0  Stream: 1  Handle: 174  Length: 234204  Speed: 100000  Pitch: 0
Aug 15 13:00:00 maxsrv rdairplay: RDLogPlay::StartEvent(): no audio,CUT=
Aug 15 13:00:00 maxsrv caed: StopPlayback - Card: 0  Stream: 0  Handle: 173
Aug 15 13:00:00 maxsrv rdairplay: finished event: Line: 410  Cart: 11707  Cut: 1 Card: 0  Stream: 0  Port: 1
Aug 15 13:00:00 maxsrv caed: UnloadPlayback - Card: 0  Stream: 0  Handle: 173
```

**Comment by ElvishArtisan** [comment]
> When a hard start time is reached in the log, and the cart does not have any playable cuts, the current playing event is stopped abruptly and the event after the hard start time event is started.

Which, for 'Start Immediately' is exactly what one wants! However, 'Make Next' is definitely broken.

**Comment by ElvishArtisan** [comment]
Fixed in [master 7b91037]. Please test!

**Comment by deltecent** [comment]
Austin confirmed that this problem is fixed.
